runtime/common/src/fixed_point.rs

```diff
@@ -134,6 +137,48 @@ macro_rules! implement_fixed {
 			fn into_inner(self) -> Self::Inner {
 				self.0
 			}
+
+			/// Creates `self` from a rational number. Equal to `n / d`.
+			///
+			/// Returns `None` if `d == 0` or `n / d` exceeds accuracy.
+			fn checked_from_rational<N: FixedPointOperand, D: FixedPointOperand>(
```

Comment:
Why did the upgrade to 0.9.26 forced us to implement this function?

Reply:
Short: Parity

Long: Parity has changed the fixed point core logic that this trait default implements on the substrate side. They choose a rounding preference, which I think they choose, as it satisfies the older already existing tests before that change.
As a plus, they now have different rounding preferences, depending on which method you use. I already opened a ticket

The possibility to choose the rounding is great though, it's just a little bit weirdly implemented on their side IMO.